datalad/distribution/install.py

```diff
@@ -323,7 +323,7 @@ def __call__(
                 # yoh: path should be a local path, and mapping note within
                 #      SSHRI about mapping localhost:path to path is kinda
                 #      a peculiar use-case IMHO
-                path = resolve_path(path_ri.localpath, dataset)
+                path = rev_resolve_path(path_ri.localpath, dataset)
```

Comment:
That is what confuses me - why then this new function should not just replace the old one? When should I use old over overwatch if the new one?

Reply:
It always has to go. I just don't have the juice to do it all at once: https://github.com/datalad/datalad/issues/3778